pkg/kubernetes/informer/informer.go

```diff
@@ -99,16 +99,6 @@ func (i *Informer) OnUpdate(cb func(oldObj, newObj interface{})) {
 	i.eventHandlers.UpdateFunc = cb
 }
 
-// OnAdd is triggered when an object is added.
-func (i *Informer) OnAdd(cb func(obj interface{})) {
-	i.eventHandlers.AddFunc = cb
-}
-
-// OnDelete is triggered when an object is deleted.
-func (i *Informer) OnDelete(cb func(obj interface{})) {
-	i.eventHandlers.DeleteFunc = cb
-}
-
```

Comment:
I would actually not remove this, they are a part of the `Informer` utility. We are currently using only `OnUpdate` for reloading the RBAC configMap, but it is possible to use `OnAdd` and `onDelete` too for other use-cases.

What I mean is this is a generic helper library to watch Kubernetes events and perform actions, so without these 2 methods, the library is kind of incomplete even though we aren't using it anywhere 😅